test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -1063,6 +1063,21 @@ ava('.maintainHeading() returns a success message when incremental is true and d
     t.deepEqual(result, expectedResult);
 });
 
+ava('.maintainHeading() returns a success message when incremental is true and direction is right, and a 1-digit numeral is used for the increment', (t) => {
+    const directionMock = 'right';
+    const expectedResult = [
+        true,
+        {
+            log: 'turn 5 degrees right',
+            say: 'turn 5 degrees right'
+        }
+    ];
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const result = aircraftModel.pilot.maintainHeading(aircraftModel, 5, directionMock, true);
+
+    t.deepEqual(result, expectedResult);
+});    
```

Comment:
Actually, lint-diff shows there are trailing spaces on this line, please remove when able :)